docs-ui/components/confirm.stories.js

```diff
@@ -9,16 +9,15 @@ import Button from 'sentry-ui/buttons/button';
 storiesOf('Confirm/Confirm', module).add(
   'default',
   withInfo({
-    text: 'Component whose child is rendered as the "action" component that when clicked opens the "Confirm Modal"',
-    propTablesExclude: [Button]
+    text:
+      'Component whose child is rendered as the "action" component that when clicked opens the "Confirm Modal"',
```

Comment:
😢

Reply:
yeah damn, using template strings forces it to be inline

Reply:
Related:
https://github.com/prettier/prettier/issues/2889  
https://github.com/prettier/prettier/issues/2482